pegasus/helpers/hourofcode_helpers.rb

```diff
@@ -110,12 +110,27 @@ def hoc_get_locale_code
 end
 
 # code.org and hourofcode.com's /learn pages call this to translate tutorial's languages attribute
-def hoc_language(lang_codes_str)
+def hoc_language(lang_codes_str, language = nil)
```

Comment:
Are we just adding the `language` parameter for testing?

Reply:
The language param is only for testing, like dependency injection.  Without it we cannot call `hoc_s`, which relies on language data from request or url path.  We should design our functions in a way that is easier to test.

Reply:
Could we add some comments above these functions explaining it for future users?